contrib/debian/changelog

```diff
@@ -1,3 +1,9 @@
+zcash (1.0.10+1) stable; urgency=medium
```

Comment:
I don't think so. I don't really know the impact of the field. From [Footnote 42 linked from here](https://www.debian.org/doc/debian-policy/ch-controlfields.html#s-f-Urgency): 

> The urgency affects how quickly a package will be considered for inclusion into the testing distribution and gives an indication of the importance of any fixes included in the upload.